framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -1184,4 +1185,27 @@ private static Range bigRangeToLongRange(BigInteger bigFrom, BigInteger bigTo) {
         long longTo = bigTo.longValue();
         return createRangeOrEverything(longFrom, longTo);
     }
+
+    /**
+     * Returns a Range representing all possible values for the given primitive type.
+     *
+     * @param typeKind is a java.lang.TypeKind that isPrimitive() and integral.
```

Comment:
You don't need to restate the Java type.  For brevity, please just give facts that aren't obvious from the method's signature.